include/enums.h

```diff
@@ -6269,41 +6269,47 @@ enum RushFlags {
-    FILE_MESSAGE_HAS_BEEN_DELETED           = 27, // has been deleted.[End]
-    FILE_MESSAGE_28                         = 28, // [End]
-    FILE_MESSAGE_COPY_FROM                  = 29, // Copy from[End]
-    FILE_MESSAGE_TO                         = 30, // to[End]
-    FILE_MESSAGE_HAS_BEEN_CREATED           = 31, // has been created.[End]
-    FILE_MESSAGE_ENTER_A_FILE_NAME          = 32, // Enter a file name![End]
-    FILE_MESSAGE_QUESTION                   = 33, // ?[End]
-    FILE_MESSAGE_PERIOD_34                  = 34, // .[End]
+    /*  0 */ FILE_MESSAGE_NONE,
+    /*  1 */ FILE_MESSAGE_SELECT_FILE_TO_START,        // Select file to start:[End]
+    /*  2 */ FILE_MESSAGE_SELECT_FILE_TO_DELETE,       // Select file to delete:[End]
+#if !VERSION_PAL
+    /*  3 */ FILE_MESSAGE_SELECT_FILE_TO_SAVE,         // Select file to save[End]
+#endif
+    /*  4 */ FILE_MESSAGE_COPY_WHICH_FILE,             // Copy which file?[End]
+    /*  5 */ FILE_MESSAGE_COPY_TO_WHICH_FILE,          // Copy to which file?[End]
+    /*  6 */ FILE_MESSAGE_NEW,                         // NEW[End]
+    /*  7 */ FILE_MESSAGE_LEVEL,                       // Level[End]
+    /*  8 */ FILE_MESSAGE_PLAY_TIME,                   // Play Time[End]
+    /*  9 */ FILE_MESSAGE_DELETE_FILE,                 // Delete File[End]
+    /* 10 */ FILE_MESSAGE_CANCEL,                      // Cancel[End]
+    /* 11 */ FILE_MESSAGE_COPY_FILE,                   // Copy File[End]
+    /* 12 */ FILE_MESSAGE_FIRST_PLAY,                  // First Play[End]
+    /* 13 */ FILE_MESSAGE_PERIOD_13,                   // .[End]
+    /* 14 */ FILE_MESSAGE_YES,                         // Yes[End]
+    /* 15 */ FILE_MESSAGE_NO,                          // No[End]
+    /* 16 */ FILE_MESSAGE_DELETE,                      // Delete[End]
+    /* 17 */ FILE_MESSAGE_OVERRIDE_TO_NEW_DATA,        // Override to New Data[End]
+    /* 18 */ FILE_MESSAGE_SAVE_OK,                     // Save OK?[End]
+    /* 19 */ FILE_MESSAGE_FILE_NAME_IS,                // File name is :[End]
+    /* 20 */ FILE_MESSAGE_PERIOD_20,                   // .[End]
+    /* 21 */ FILE_MESSAGE_OK,                          // OK?[End]
+    /* 22 */ FILE_MESSAGE_FILE_22,                     // File[End]
+    /* 23 */ FILE_MESSAGE_WILL_BE_DELETED,             // will be deleted.[End]
+    /* 24 */ FILE_MESSAGE_OK_TO_COPY_TO_THIS_FILE,     // OK to copy to this file?[End]
+    /* 25 */ FILE_MESSAGE_START_GAME_WITH,             // Start game with[End]
+    /* 26 */ FILE_MESSAGE_FILE_26,                     // File[End]
+    /* 27 */ FILE_MESSAGE_HAS_BEEN_DELETED,            // has been deleted.[End]
+    /* 28 */ FILE_MESSAGE_28,                          // [End]
+    /* 29 */ FILE_MESSAGE_COPY_FROM,                   // Copy from[End]
+    /* 30 */ FILE_MESSAGE_TO,                          // to[End]
+    /* 31 */ FILE_MESSAGE_HAS_BEEN_CREATED,            // has been created.[End]
+#if VERSION_PAL
+    UNK1,
```

Comment:
`FILE_MESSAGE_ENTER_A_FILE_NAME` is at the right place, with this define, but I don't know where exactly UNK1 and UNK2 should be placed

Reply:
We need to split the PAL messages to figure this out I think

Reply:
No, all filemenu messages are (unfortunately) stored in `filemenu_msg.c`. Issue is that they're in the PM encoding, so they're not easy to read.